masterlist.yaml

```diff
@@ -12366,6 +12369,13 @@ plugins:
         util: 'SSEEdit v4.0.3'
       - crc: 0xAC83F041
         util: 'SSEEdit v4.0.3'
+        
```

Comment:
I'll just remove that newline, plugins from the same mod page can be kept together.
```suggestion
```